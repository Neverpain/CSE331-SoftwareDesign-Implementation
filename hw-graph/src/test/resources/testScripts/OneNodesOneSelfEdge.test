# Tests that a graph can be created, a node can be
# added to it, a edge pointing to itself can be added,
# and the node and its edge can be printed out

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Add an edge
AddEdge graph1 n1 n1 n1n1

# Prints the nodes contained in the graph and their edges
ListNodes graph1
ListChildren n1