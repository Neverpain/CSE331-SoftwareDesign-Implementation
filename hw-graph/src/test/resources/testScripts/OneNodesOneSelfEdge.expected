# Tests that a graph can be created, a node can be
# added to it, a edge pointing to itself can be added,
# and the node and its edge can be printed out

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# Add an edge
added edge n1n1 to n1

# Prints the nodes contained in the graph and their edges
graph1 contains: n1
the children of n1 in graph1 are: n1